kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -122,7 +123,8 @@
                 :disabled="false"
                 :hasIcons="true"
                 :options="activeSectionActions"
-                @tab="e => (e.preventDefault() || $refs.selectAllCheckbox.focus())"
+                @tab="$refs.addQuestionsButton.$el.focus()"
+                @close="$refs.addQuestionsButton.$el.focus()"
```

Comment:
Why @tab & @close both doing the same?

- When closing by any means other than selecting an option, need to direct focus.

- Need to verify that `@tab` should close it?